core/common/src/test/java/alluxio/collections/IndexedSetTest.java

```diff
@@ -53,11 +53,21 @@ public long longValue() {
   @Before
   public void before() {
     mIntIndex = new IndexedSet.FieldIndex<Pair>() {
+      @Override
+      public Boolean isUnique() {
```

Comment:
+1 for having two classes. And in that case, we may not even need to have `isUnique` but we can directly use `instanceof` to tell if this is a unique field index or not.